pkg/config/config.go

```diff
@@ -17,20 +17,24 @@ type Serialized struct {
 		Timeout      string `json:"timeout"`
 		MinRetryTime string `json:"min_retry"`
 	} `json:"pull_report"`
-	Impersonate string   `json:"impersonate"`
-	Gather      []string `json:"gather"`
+	Impersonate                  string   `json:"impersonate"`
+	Gather                       []string `json:"gather"`
+	DisabledGlobalAnonymizations []string `json:"disabledGlobalAnonymizations"`
```

Comment:
If I understand it correctly, this slice defines what types of anonymization we want. I don't think it's necessary. I would simply introduce one flag saying obfuscation enabled/disabled (covering all - IPs and hostnames) but let's double check.